src/codeu/chat/client/commandline/Chat.java

```diff
@@ -107,11 +110,36 @@ public void invoke(Scanner args) {
         System.out.println("    Add a new user with the given name.");
         System.out.println("  u-sign-in <name>");
         System.out.println("    Sign in as the user with the given name.");
+        System.out.println("  uptime");
+        System.out.println("    Display the amount of time the server has been running.");
         System.out.println("  exit");
         System.out.println("    Exit the program.");
       }
     });
 
+    //UPTIME
+    //
+    // Add a command to get the server uptime when the user enters "uptime"
+    // while on the root panel.
+    //
+    panel.register("uptime", new Panel.Command() {
+      @Override
+      public void invoke(Scanner args) {
+        final ServerInfo info = context.getServerUptime();
+        if(info == null)
```

Comment:
Some style nits (mostly based on [the Google Java Style Guide](https://google.github.io/styleguide/javaguide.html), which I do see linked from the [CodeU Code Review Overview doc](https://docs.google.com/document/d/19Cfv1bzdJfzIqgz9R87RSQzjKuGAKOj0SvhxjtOmg9o/edit) and so I think I'm supposed to point this out):

- Please put a space between keywords (in this case, `if`) and the following expression.
- Especially because the `else` has curly-brackets around its block, please put curly-brackets around the `if` block as well.